src/components/_filters/ChainFilter.tsx

```
@@ -72,25 +72,32 @@ export const ChainFilter = (props: ChainFilterProps) => {
+      <Avatar
+        name={chain.displayName}
+        src={chain.logoPath}
+        key={chain.id}
+        background="transparent"
+        border="none"
+        size="sm"
+        marginLeft="-0.5rem"
+        _last={{ marginLeft: 1 }}
+      />
+    )
+  })
```

ESLint

[error] Component definition is missing display name

(react/display-name)